coding-challenges/tip-calculator/script.js

```diff
@@ -0,0 +1,38 @@
+document.addEventListener("DOMContentLoaded", function () {
+    const calculateButton = document.getElementById("calculateBtn");
+    const clearButton = document.getElementById("clearBtn");
+
+    calculateButton.addEventListener("click", calculateTip);
+    clearButton.addEventListener("click", clearFields);
+});
+
+function calculateTip() {
+    let price = parseFloat(document.getElementById("price").value);
+    let tipPercentage = parseFloat(document.getElementById("tipPercentage").value);
+    let numPeople = parseInt(document.getElementById("numPeople").value);
```

Comment:
I am all for not overusing global variables.
However, if we are constantly querying the DOM for the same elements, then it does make sense to make those global variables so they can be reused in multiple places.
So I would suggest elevating those variables as well as the other places this happens to global variables.